applications/FluidDynamicsApplication/tests/adjoint_vms_sensitivity_2d.py

```diff
@@ -0,0 +1,198 @@
+import os
+from KratosMultiphysics import *
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics.FluidDynamicsApplication
+
+missing_applications_message = ["Missing required application(s):",]
+have_required_applications = True
+
+try:
+    import KratosMultiphysics.AdjointFluidApplication
+except ImportError:
+    have_required_applications = False
+    missing_applications_message.append("AdjointFluidApplication")
+
+try:
+    import KratosMultiphysics.HDF5Application as kh5
+except ImportError:
+    have_required_applications = False
+    missing_applications_message.append("HDF5Application")
+
+from fluid_dynamics_analysis import FluidDynamicsAnalysis
+
+if have_required_applications:
+    from adjoint_fluid_analysis import AdjointFluidAnalysis
+
+class ControlledExecutionScope:
+    def __init__(self, scope):
+        self.currentPath = os.getcwd()
+        self.scope = scope
+
+    def __enter__(self):
+        os.chdir(self.scope)
+
+    def __exit__(self, type, value, traceback):
+        os.chdir(self.currentPath)
+
+@KratosUnittest.skipUnless(have_required_applications," ".join(missing_applications_message))
+class AdjointVMSSensitivity2D(KratosUnittest.TestCase):
+
+    def setUp(self):
+        pass
```

Comment:
There is a lot of print out in these tests coming from the `_ImportModelPart()` of the `python_solver.py`. What about modifying the severity level in the `setUp` and `tearDown` like the following?

```
def setUp(self):
        self._logger_severity = KratosMultiphysics.Logger.GetDefaultOutput().GetSeverity()
        KratosMultiphysics.Logger.GetDefaultOutput().SetSeverity(KratosMultiphysics.Logger.Severity.WARNING)

def tearDown(self):
        KratosMultiphysics.Logger.GetDefaultOutput().SetSeverity(self._logger_severity)
```

Reply:
This will not filter out the std::cout prints which is still a lot of outputs. I'm not sure how to get rid of these printouts except for using -v2 as an argument when testing.

Reply:
Since, this PR is massive, Will it be ok if I add these changes in a separate PR?

Reply:
ok

Reply:
Which couts are left?
The tests in StructuralMechanics dont print anything any more if the lvl of severity is increased